### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by petersteneteg**

### CODE REVIEW

File: modules/animationqt/animationviewqt.cpp [hunk 1]
```diff
@@ -42,6 +42,7 @@
 #include <QGridLayout>
 #include <QScrollBar>
 #include <QGraphicsScene>
+#include <QStyleOption>
```
Comment (martinfalk) [inline modules/animationqt/animationviewqt.cpp:45]: probably don't need this?

File: modules/qtwidgets/properties/syntaxhighlighter.cpp [hunk 2]
```diff
@@ -64,8 +64,8 @@ template <>
 void SyntaxHighligther::loadConfig<None>() {
     auto ivec4toQtColor = [](const ivec4& i) { return QColor(i.r, i.g, i.b, i.a); };
     auto settings = InviwoApplication::getPtr()->getSettingsByType<QtWidgetsSettings>();
-    QColor textColor = ivec4toQtColor(settings->glslTextColor_.get());
-    backgroundColor_ = ivec4toQtColor(settings->glslBackgroundColor_.get());
+    QColor textColor = ivec4toQtColor(settings->pyTextColor_.get());
+    backgroundColor_ = ivec4toQtColor(settings->pyBGColor_.get());
```
Comment (martinfalk) [inline modules/qtwidgets/properties/syntaxhighlighter.cpp:68]: Do you know why this is called `pyTextColor_`? Isn't this a global color for the syntax highlighter?

**Review by martinfalk — APPROVED** [review]

File: modules/qtwidgets/properties/syntaxhighlighter.cpp (hunk 2 above)
Comment (petersteneteg) [reply modules/qtwidgets/properties/syntaxhighlighter.cpp:68]: There is a py and a glsl version...

File: modules/animationqt/animationviewqt.cpp (hunk 1 above)
Comment (petersteneteg) [reply modules/animationqt/animationviewqt.cpp:45]: True

**Pull request merged**